code/game/objects/items/devices/flashlight.dm

```diff
@@ -320,6 +315,8 @@
 		user.visible_message("<span class='notice'>[user] lights \the [src].</span>", "<span class='notice'>You light \the [src]!</span>")
 		force = on_damage
 		damtype = "fire"
+		add_emitter(/obj/emitter/sparks/flare, "spark", 10)
+		add_emitter(/obj/emitter/flare_smoke, "smoke", 9)
```

Comment:
This incorrectly adds these emitters to the '/obj/item/flashlight/flare/torch' as well, the smoke and sparks are in the wrong location.

Also, the sparks are in the wrong location in general as well.

Reply:
on the torch version? the normal flare looked well on local